packages/coinstac-simulator/src/boot-clients.js

```diff
@@ -1,48 +1,77 @@
 'use strict';
 
-/**
- * @private
- * @module boot-clients
- */
+require('./utils/handle-errors');
 
-require('./handle-errors')();
 const cp = require('child_process');
-const chalk = require('chalk');
+const logger = require('./utils/logger');
 const path = require('path');
-const logger = require('./logger');
-const userChatOK = chalk.magenta;
 
-module.exports = function bootClients(declPath) {
+/**
```

Comment:
If you dont mind, and you understand these funcs well, mind writing some more explanation in the docblocs for future generations?